index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      href="https://fonts.googleapis.com/css?family=Poppins"
+      rel="stylesheet"
+    />
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+  <body>
+    <header>
+      <nav>
+        <a href="#">My Logo</a>
+        <img id="menu-icon" src="images/Enabled.png" alt="" />
+      </nav>
+    </header>
+
+    <main>
+      <h1>I'm Shimels, Glad to see you!</h1>
+      <p>
+        Full-stack developer with over 3 years of experience in developing web
+        applications. Proficient in both front-end and back-end technologies,
+        including Angular, ASP.NET Core Web API, Node.js, Flask, and SQL.
+        Skilled in troubleshooting and maintaining databases and server-side
+        infrastructure. Adept at analytical thinking and problem-solving.
+        Excellent communication skills and customer focus.
+      </p>
+      <span> Let's Connect</span>
+      <section id="social-links">
+        <a href="#"> <img src="icones/icons8-twitter-50.png" alt="" /></a>
+        <a href="#"> <img src="icones/icons8-linkedin-50.png" alt="" /></a>
+        <a href="#"> <img src="icones/icons8-github-50.png" alt="" /></a>
+      </section>
```

Comment:
- [ ] Also as mentioned by the last reviewer, your social links are missing some icons. I'll suggest you make sure that all the icons in the Figma design are in your project and arranges them accordingly. :+1: 

- - Your design:
![Screenshot from 2023-07-05 15-36-45](https://github.com/shimels1/portfolio_microverse/assets/107632780/2fc937a4-d061-49d5-ac24-8254004e3a9a)

- - Figma design:
![Screenshot from 2023-07-05 15-36-29](https://github.com/shimels1/portfolio_microverse/assets/107632780/0b383d59-7b68-4a49-8451-8ff55939911c)

Reply:
- [ ] As requested by the previous reviewer, kindly add the missing icons as they appear in the Figma. Practice producing designs just as they are given to you because this is na important skill on a job 👏🏾